controllers/commentAnswer.js

```diff
@@ -0,0 +1,33 @@
+const models = require('../models/index');
+const commentAnswer = async (req, res) => {
+    try {
+        const commentId = parseInt(req.params.id);
+        const { comment_text, answer_id } = req.body;
+        const user_id = req.user_id;
+
+        //check if the answer exists
+        const answer = await models.answers.findOne({
+            where: {
+                id: answer_id,
+            },
+        });
+
+        if (answer) {
+            const comment = await models.comments.create({
+                comment_text: comment_text,
+                id: commentId,
```

Comment:
Remove this line. When creating a record, the ID is auto-increment meaning, the database system will insert the right value automatically so users don't need to except `Auto-Increment` was explicitly disabled.  

<img width="1187" alt="Screenshot 2022-09-30 at 11 49 07 AM" src="https://user-images.githubusercontent.com/82178278/193254520-1445830f-e67b-4c7b-83e4-a36550c870ea.png">